src/routes/docs/products/auth/teams/+page.markdoc

```diff
@@ -187,6 +187,35 @@ val response = teams.createMembership(
 
 {% /multicode %}
 
+# CLI {% #CLI %}
+
+{% info title="CLI setup" %}
+Before you can deploy with the Appwrite CLI, make sure you've [installed and initialized](/docs/tooling/command-line/installation) the CLI and [logged-in](/docs/tooling/command-line/installation#get-started) into your Appwrite account.
```

Comment:
According to this [reference](https://ell.stackexchange.com/questions/167389/the-user-is-logged-in-vs-the-user-is-logged-in#:~:text=The%20second%20one%20is%20incorrect,the%20logged%2Din%20user%22.) I found online, you are correct that "logged-in" should not be hyphenated because it is not being used as adjective, but rather a verb in this sentence.